sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample_AnalyzeWithCustomModelFromFileAsync.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.DocumentAnalysis.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis.Samples
+{
+    public partial class DocumentAnalysisSamples : SamplesBase<DocumentAnalysisTestEnvironment>
+    {
+        [Test]
+        public async Task AnalyzeWithCustomModelFromFileAsync()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            Uri trainingFileUri = new Uri(TestEnvironment.BlobContainerSasUrl);
+
+            // Firstly, create a custom built model we can use to recognize the custom document. Please note
+            // that models can also be built using a graphical user interface such as the Form Recognizer
+            // Labeling Tool found here:
+            // https://docs.microsoft.com/azure/cognitive-services/form-recognizer/label-tool?tabs=v2-1
```

Comment:
not sure if this is public but should probably switch to the aka.ms link we created for labeling tool

Reply:
great find. yes! the new link is => https://aka.ms/azsdk/formrecognizer/labelingtool